addons/test_mail/tests/test_mail_activity.py

```diff
@@ -333,3 +333,43 @@ def test_activity_mixin_reschedule_user(self):
             user_id=self.user_admin.id,
             new_user_id=self.user_employee.id)
         self.assertEqual(rec.activity_ids[0].user_id, self.user_employee)
+
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_my_activity_date_deadline(self):
```

Comment:
I suggest to use existing data (users, test records) instead of re-creating everything here. It lessens boilerplate and allow to have smaller tests :) .